libc/test/src/time/clock_gettime_test.cpp

```diff
@@ -15,8 +16,14 @@ TEST(LlvmLibcClockGetTime, RealTime) {
   struct timespec tp;
```

Comment:
Since you are here, can you remove the redundant `struct` also?

Reply:
```suggestion
```